mozci/task.py

```diff
@@ -218,6 +222,14 @@ def to_json(self):
         sig = signature(self.__init__)
         return {k: v for k, v in self.__dict__.items() if k in sig.parameters}
 
+    def retrigger(self):
```

Comment:
Please add a docstring for this function that defines what it does, this will ensure it gets listed in the `mozci` API documentation :)